.gitignore

```diff
@@ -22,4 +22,4 @@ pubspec.lock
 Podfile.lock
 
 .gradle
-flutter/.gradle
+packages/flutter/.gradle
```

Comment:
Can't comment above, but the melos docs says we should exclude the lock files:

```gitignore
pubspec.lock
```

> The corresponding pubspec.lock file should also be committed. Make sure to exclude it from the .gitignore file.

https://melos.invertase.dev/getting-started#install-melos-in-the-workspace

Is this releavant to our structure?

Reply:
not sure, I don't think it applies to libraries

it would make sense for normal end apps